### PR TITLE
Correct the duration for training and validation set in documentation ("Prevent Overfitting" Session)

### PR DESCRIPTION
**Pull request by riven314**

### CODE REVIEW

**Comment by saleh-mir** [comment]
Thank you
